app/assets/stylesheets/onboarding.scss

```diff
@@ -0,0 +1,66 @@
+@import 'variables';
+@import 'mixins';
+@import 'config/import';
+
+.onboarding-task-card {
+  background: #3b49df;
+  border-radius: $br-default;
+  padding: $su-7 $su-8 $su-8 $su-8;
+  display: none;
+
+  .task-card-title {
+    font-weight: $fw-heavy;
+    font-size: $fs-4xl;
+    line-height: $lh-tight;
+    color: #ffffff;
+  }
+
+  .task-card-subtitle {
+    color: #ffffff;
+    opacity: 0.7;
+    font-weight: $fw-medium;
+    font-size: $fs-s;
+    line-height: $lh-tight;
+    letter-spacing: -0.02em;
+  }
+
+  .task-card-action {
+    border-radius: $br-default;
+    background-color: rgba(255, 255, 255, 0.2);
```

Comment:
i feel like we should be using a crayons color here, too? i'm sure either @ludwiczakpawel or @lisasy could confirm if crayons handles opacity. if we absolutely _have_ to use `rgba` here, i think it warrants a comment. feels weird to see this in the file given the fact that we're defaulting to using hex or crayons...

Reply:
I initially had concerns with this too then I notice the figma spec provided `background-color: rgba(fff,fff,fff,0.2)` as a guideline. I tweak `fff` to 255 because it wasn't valid.

This on-hover effect can't be produced with combination use of hex and opacity because the opacity will affect the text within it.

Reply:
Okay, let's just leave a comment about it then so that someone doesn't accidentally change this in the future.

Reply:
Added